src/pages/resource/ResourceUsage.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { FC, useEffect, useRef, useState } from 'react';
+import { Resource } from '../../types/resource.types';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import { format } from 'date-fns';
+import { TFunction, useTranslation } from 'react-i18next';
+import { getCitationFromCrossCite } from '../../api/resourceApi';
+import styled from 'styled-components';
+
+const StyledTextAndButtonWrapper = styled.div`
+  display: block;
+  margin-top: 2rem;
+`;
+
+const StyledInformationWrapper = styled.div`
+  display: flex;
+  @media (max-width: ${({ theme }) => theme.breakpoints.values.sm + 1 + 'px'}) {
+    display: block;
+  }
+`;
+
+const StyledTypography = styled(Typography)`
+  @media (min-width: ${({ theme }) => theme.breakpoints.values.sm + 'px'}) {
+    margin-right: 2rem;
+  }
+`;
+
+const generateCitationStringFromResourceMetadata = (resource: Resource, t: TFunction<'translation'>): string => {
+  let citation = resource.creators
+    .sort((creatorA, creatorB) => {
```

Comment:
kan sikkert bruke sortCreatorArray fra creatorfield

Reply:
Fikset

Reply:
Forøvrig er det litt legacy at creator order blir tatt hensyn til. Under utvikling av creator-feltene så blir alle creatorer satt til order = 0, er kun ressurser i prod på gamle DLR som har ulike order

Reply:
Da kunne vi kanskje bare droppet hele sjekken, om den kun er tiltenkt BIRD.